### PR TITLE
Unique identifier seen by internal users in the "Request ID" field in Operator table

### PR DESCRIPTION
**Pull request by marcellmueller**
Implements #1037 

Creating a sequence in Django was challenging since Django only allows one `AutoField` since it has to have `primary_key=true` set to work. Thanks to help from Sepehr we have come up with our own solution.

To test that this will work with User Operator rows that are already in prod, you can do the following steps:
```
git checkout develop
cd /bc_obps && make reset_db
git checkout 1037-user-operator-user-friendly-unique-id
make migrate
psql -d registration
select user_friendly_id from erc.user_operator;
```
You should see the fixtures we set up in the develop branch now have a `user_friendly_id` column with the user friendly ids added:

Screenshot 2024-03-14 at 2 29 09 PM

Screenshot 2024-03-14 at 3 01 35 PM

### CODE REVIEW

File: bc_obps/registration/fixtures/mock/userOperator.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
       "role": "admin",
       "status": "Approved",
       "verified_at": "2024-02-26 06:24:57.293242-08",
-      "verified_by": "ba2ba62a121842e0942aab9e92ce8822"
+      "verified_by": "ba2ba62a121842e0942aab9e92ce8822",
+      "user_friendly_id": 1
```
Comment (marcellmueller) [inline bc_obps/registration/fixtures/mock/userOperator.json:11]: Needed to add this to the fixtures as the fixtures are loaded after the migration so the function that adds this to existing user operators won't run on these.

File: bc_obps/registration/models.py [hunk 2]
```diff
@@ -19,6 +19,7 @@
 from django.core.exceptions import ValidationError
 from django.core.files.storage import default_storage
 from django.core.cache import cache
+from django.utils.translation import gettext as _
```
Comment (Sepehr-Sobhani) [inline bc_obps/registration/models.py:22]: Leftover!

**Review by Sepehr-Sobhani — APPROVED** [review]
Just a redundant import otherwise looks great, Thanks! 🙏

File: bc_obps/registration/models.py (hunk 2 above)
Comment (marcellmueller) [reply bc_obps/registration/models.py:22]: Nice catch, thanks 🙇

**Pull request merged**